pages/gallery.html

```diff
@@ -4,9 +4,54 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Gallery</title>
+  <link href="../styles/style.css" rel="stylesheet"> 
+  <link href="../styles/gallery.css" rel="stylesheet">
 </head>
 <body>
-  
+  <header>
+    <h1>Memes</h1>
+    <nav>
+      <ul>
+        <li>
+          <a href="../index.html">Home</a>
+        </li>
+        <li>
+          <a href="facts.html">Fun Facts</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <div class="container">        
+      <div class="card">
+        <img alt="Meme 1: programming languages hierarchy" src="../images/1.jpg">
+        <a href="https://www.reddit.com/r/ProgrammerHumor/comments/bufykz/c_java_python_student/">source</a>
+      </div>
```

Comment:
Prefer a more semantic tag over a generic `div` tag for each of these elements with the "card" class. 

Perhaps [`figure`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/figure) could work: "The figure element represents self-contained content, potentially with an optional caption, which is specified using the [<figcaption>](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/figcaption) element. The figure, its caption, and its contents are referenced as a single unit."

```suggestion
      <figure class="card">
        <img alt="Meme 1: programming languages hierarchy" src="../images/1.jpg">
        <figcaption>
          <a href="https://www.reddit.com/r/ProgrammerHumor/comments/bufykz/c_java_python_student/">source</a>
        </figcaption>
      </figure>
```